src/default-xpring-client.ts

```diff
@@ -163,7 +163,8 @@ class DefaultXpringClient implements XpringClientDecorator {
   }
 
   public async getLastValidatedLedgerSequence(): Promise<number> {
-    throw new Error(XpringClientErrorMessages.unimplemented)
+    const getFeeResponse = await this.getFee()
```

Comment:
We need to get the fee in order to get the validated ledger sequence?

Reply:
Yes. The `Fee` rpc happens to contain this information and we only implemented the minimal subset of these APIs in rippled to make this happen so this is what we're using for now.